src/deluge/gui/views/instrument_clip_view.cpp

```diff
@@ -2170,7 +2171,7 @@ void InstrumentClipView::adjustProbability(int32_t offset) {
 		// Decide the probability, based on the existing probability of the leftmost note
 		probabilityValue = editPadPresses[leftMostIndex].intendedProbability & 127;
 		probabilityValue += offset;
-		probabilityValue = std::clamp<int32_t>(probabilityValue, 1, kNumProbabilityValues + 35);
+		probabilityValue = std::clamp<int32_t>(probabilityValue, 1, kNumProbabilityValues + kNumIterationValues);
```

Comment:
Is it possible to have a probability of 0? If not please change line 2100 to be >=1 instead of >0

Reply:
I have repurposed 0 as the fill value. I like 0 for fill because [1] it's the cleanest way to implement with the least code disruption and [2] conceptually the fill notes *do* have a zero percent chance of playing during normal playback.